### DISCUSSION TITLE
Get more information about vegetation

### DISCUSSION THREAD

**Discussion by j3r3m1** [orbisgis/geoclimate#843] [Ideas]
In OSM, forest, wood and single trees might have a tag about:
- the leave type via the tag [leaf_type](https://wiki.openstreetmap.org/wiki/Key:leaf_type)
- the phenology of the tree via the tag [leaf_cycle](https://wiki.openstreetmap.org/wiki/Key:leaf_cycle)

I suppose this information might be useful for climate modelling, what do you think ?
@wurtzj @ebocher

**Comment by wurtzj** [comment]
Hi @j3r3m1 ,

Seems useful to me :)
